tests/unit/test_base_command.py

```diff
@@ -17,10 +17,13 @@
 
 @pytest.fixture
 def fixed_time() -> Iterator[None]:
-    with patch("time.time", lambda: 1547704837.040001 + time.timezone):
+    with patch("time.time", lambda: 1547704837.040001):
         yield
 
 
+logging.Formatter.converter = time.gmtime
```

Comment:
Should this be in a fixture instead? Modifying the global state in test seems wrong to me.